lib/config/executor/container_vm.go

```diff
@@ -34,7 +34,7 @@ type Common struct {
 	// A reference to the components hosting execution environment, if any
 	ExecutionEnvironment string
 
-	// Unambiguous ID with meaning in the context of its hosting execution environment
+	// Unambiguous ID with meaning in the context of its hosting execution environment. Change this definition will cause container backward compatibility issue
```

Comment:
please add "Please don't change this." I am not sure, you might add a "without talking to emma lin" to that. That way if someone does want to change it for  whatever reason they can talk to you about why they likely shouldn't.